akka-operator/Chart.yaml

```diff
@@ -6,6 +6,6 @@ description: Akka Operator in Akka Cloud Platform
 # Akka Operator
 type: application
 # This is the chart version, which corresponds to the version of the Akka Operator.
-version: 0.4.5
+version: 0.4.7
 # This is the version number of the Akka Operator, being deployed.
-appVersion: 0.4.5
+appVersion: 0.4.7
```

Comment:
latest version is 1.0.1
so this isn't update automatically when we release by tag?

Reply:
Patrik - I am testing stuff now and the deployer at `akka-cloud-platform-gcp` pulls from this repository. Hence I am creating some temporary versions to enable the deployer to pull from this repo.

Reply:
The release workflow should be updating version.  You can see it [updated it in this commit](https://github.com/lightbend/akka-operator-helm/commit/398413f3e88fdb2346fe743ba861c80b2c3f3635#diff-a0380d4e6fc1fab34a0a54dae8adb5b69f5dc41a1708dcd11613bb9c4d02f033), but then a following commit overwrote it with an earlier snapshot version.  Maybe someone was manually testing the release process.

Reply:
@seglo I was testing the gcp deployer stuff with the metering agent. The deployer at `akka-cloud-platform-gcp` pulls from the repo `akka-helm-operator` - hence I had to commit and push. Though I am working on the branch.

Reply:
Makes sense. It only really affects someone who installs the chart from this repo's source so we can clean it up later or just wait for a new tag.